docs/source/actor-rhel7-to-rhel8.md

```diff
@@ -15,7 +15,7 @@ The main tools you will use for the actor development are listed below.
 
 ### leapp
 
-The leapp framework uses provides the libraries required to be imported by any actor and also a binary tool used to control the execution of actors within a workflow.  
+The leapp framework uses provides the libraries required to be imported by any actor and also a binary tool used to control the execution of actors within a workflow.
```

Comment:
I know this was not done but you but we can fix it here.

> uses provides the libraries

Reply:
apart from this looks good to me...

Reply:
`The leapp framework provides the libraries` will do?

Reply:
I guess the original idea was: `The leapp framework uses the provided libraries ...`.  But @fernflower seems better to me.

Reply:
@fernflower sounds good...